.travis.yml

```diff
@@ -1,7 +0,0 @@
-sudo: false
-language: node_js
-node_js:
-  - "v15.8.0"
-script:
-  - commitlint-travis --config commitlint.config.json
```

Comment:
Linting commits in CI is given up on in this PR. Not a big deal, because a pre-commit hook should have already run the linting in most cases. And commit messages should be reviewed manually anyway.